### PR TITLE
Adds node configuration of each node in detailed view(non-editable)

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Adds Node Configuration to detailed view
- Removes Schema on Edge in detailed view (as the schema information is not consistently available in UI)
- Fixes hydrator and cdap header to highlight appropriate options.

nodeconfig-detailed

### CODE REVIEW

File: cdap-ui/app/features/adapters/controllers/create/plugin-edit-form-ctrl.js [hunk 1]
```diff
@@ -156,6 +159,7 @@ angular.module(PKG.name + '.feature.adapters')
         return null;
       }
     }
+    /////////////////////////////////////
```
Comment (anwar6953) [inline cdap-ui/app/features/adapters/controllers/create/plugin-edit-form-ctrl.js]: remove?

**Comment by anwar6953** [comment]
LGTM

**Comment by elfenheart** [comment]
:+1: LGTM on @ajainarayanan changes

**Comment by ajainarayanan** [comment]
+:1: lgtm on @elfenheart's changes

**Pull request merged**